device/core/src/module_client.ts

```diff
@@ -290,12 +370,14 @@ export class ModuleClient extends InternalClient {
         callback(err);
       } else {
         const transport = new transportCtor(authenticationProvider);
-
         // Codes_SRS_NODE_MODULE_CLIENT_13_035: [ If the client is running in edge mode then the IotEdgeAuthenticationProvider.getTrustBundle method shall be invoked to retrieve the CA cert and the returned value shall be set as the CA cert for the transport via the transport's setOptions method passing in the CA value for the ca property in the options object. ]
         transport.setOptions({ ca });
 
+        const methodClient = new MethodClient(authenticationProvider);
+        methodClient.setOptions({ ca });
```

Comment:
Shouldn't this CA get set for the `MethodClient` from the `_fromEnvironmentNormal` function too?

Reply:
aren't methods supported only in the Edge environment?

---
In reply to: [197007661](https://github.com/Azure/azure-iot-sdk-node/pull/322#discussion_r197007661) [](ancestors = 197007661)

Reply:
my bad, my misunderstanding. I was confused and thought the _fromEnvironmentNormal method was for non-edge environment (iot hub). it actually refers to env variables as opposed to edgelet. will fix.

---
In reply to: [197012544](https://github.com/Azure/azure-iot-sdk-node/pull/322#discussion_r197012544) [](ancestors = 197012544,197007661)

Reply:
Yeah, I tried finding _fromEnvironmentAbnormal, but couldn't. :)

---
In reply to: [197197439](https://github.com/Azure/azure-iot-sdk-node/pull/322#discussion_r197197439) [](ancestors = 197197439)